### PR TITLE
Fix broken `paragdiwan.gitpatch` extension build

### PR DESCRIPTION
**Pull request by manuth**
- [x] I have read the note above about PRs contributing or fixing extensions
- [x] ~~I have tried reaching out to the extension maintainers about publishing this extension to OpenVSX (if not, please create an issue in the extension's repo using [this template](https://github.com/open-vsx/publish-extensions/blob/HEAD/docs/external_contribution_request.md)).~~
- [x] This extension has an [OSI-approved OSS license](https://opensource.org/licenses) (we don't accept proprietary extensions in this repository)

## Description

Changes made in this PR will update the commit reference to build `paragdiwan.gitpatch` from as the latest version doesn't build properly.
Furthermore, changes made in this PR will fully implement the `checkout`-feature mentioned in this repository's `README`.

### CODE REVIEW

**Comment by filiptronicek** [comment]
Hi there @manuth! I am happy that you are fixing this extension, but I don't think because of it, there is enough purpose to introduce a brand-new option for it. It still sadly doesn't build even with this introduced. I think we should remove the extension from here and ask the author to publish it themselves.

**Comment by manuth** [comment]
@filiptronicek thanks for your rapid answer!
I wasn't aware that the `"checkout"`-feature described in the `README` wasn't implemented, so I added it with my last commit (
39c048f).

Hope you don't mind me introducing this change.

**Comment by filiptronicek** [comment]
> I wasn't aware that the "checkout"-feature described in the README wasn't implemented, so I added it with my last commit 

Oh, it is because of an outdated README. We have deprecated the checkout property and no longer use it.

**Comment by manuth** [comment]
> > I wasn't aware that the "checkout"-feature described in the README wasn't implemented, so I added it with my last commit
> 
> Oh, it is because of an outdated README. We have deprecated the checkout property and no longer use it.

Oh that's fair, I wasn't aware of that.
I guess once the `custom`-feature is established, having a `checkout`-property is not necessary anymore anyways.

Should I put this PR on hold until the `custom`-feature has been established?

**Comment by filiptronicek** [comment]
> Should I put this PR on hold until the custom-feature has been established?

Sounds like a good idea to me :)

**Comment by filiptronicek** [comment]
@manuth just merged the custom property PR, so feel free to use it :)

**Comment by manuth** [comment]
Awesome! Thanks for letting me know 😄

**Comment by manuth** [comment]
Oh... well that's weird, the GitHub actions won't run 😅

File: extensions.json [hunk 1]
```diff
@@ -944,7 +944,13 @@
     "repository": "https://github.com/paiqo/Databricks-VSCode"
   },
   "paragdiwan.gitpatch": {
-    "repository": "https://github.com/paragdiwan/vscode-git-patch"
+    "repository": "https://github.com/paragdiwan/vscode-git-patch",
+    "location": ".",
+    "custom": [
+      "git checkout a648bb3656d0c5c9e9c8cb281eb71d56fd0e9f6a",
+      "npm i",
+      "vsce package --out extension.vsxi"
```
Comment (filiptronicek) [inline extensions.json:952]: @manuth I think there is a typo with `.vsix`

Comment (manuth) [reply extensions.json:952]: Oh - thanks for pointing this out 😄

**Comment by manuth** [comment]
I'll close this PR in favor of #565

**Pull request closed without merging**